spec/arel/sql_to_arel_spec.rb

```diff
@@ -102,6 +98,13 @@
         sql_to_arel: false
   visit 'sql', 'EXPLAIN SELECT 1', pg_node: 'PgQuery::EXPLAIN_STMT', sql_to_arel: false
   visit 'sql', 'FETCH some_cursor', pg_node: 'PgQuery::FETCH_STMT', sql_to_arel: false
+  visit 'sql',
+        'PREPARE some_plan (integer) AS (SELECT $1)',
```

Comment:
Could you add more specs, which also test:
- no arguments for the PREPARE
- DEALLOCATE ALL (https://www.postgresql.org/docs/10/sql-deallocate.html)